### DISCUSSION TITLE
Digitaler Wasserzähler mit kursive Ziffern

### DISCUSSION THREAD

**Discussion by schnuller123** [jomjol/AI-on-the-edge-device#2807] [Q&A]
Hallo
Ich habe einen neuen Digitalen Wasserzähler und der hat kursive Zahlen
ich kommen nicht weiter!
Sollte ich mal eine andere Class verwenden, bisher verwende ich 
dig-cont_0620_s3_q.tflite

im log steht:
[0d00h06m03s] 2024-01-13T14:39:25 <WRN> [CNN] 
Value Rejected due to Threshold (Fit: 0.261719, Threshold: 0.500000)

[0d00h06m04s] 2024-01-13T14:39:25 <DBG> [CNN] 
Processing Number 'AbsolutMenge'

es wird mir nur:
Process State
Initialization (delayed)
angezeigt.

/
![Bild](https://github.com/jomjol/AI-on-the-edge-device/assets/132897902/3bdf34af-c58d-4504-9897-75921c679ffc)

was mach ich falsch?

**Comment by caco3** [comment]
Suche mal in den Diskussionen danach.
Du kannst das kursif-sein etwas kompensieren, idem du das Bild etwas gegen den Uhrzeigersinn drehst. Natürlich dürfen die ROIs dann nicht mehr den gleichen Y-Wert haben.
